Makefile

```diff
@@ -0,0 +1,13 @@
+
+all: build-conf
+
+.PHONY: build-conf docker clean
+
+build-conf:
+	node build-nginx-configs.js
+
+docker: build-conf
+	docker-compose build
+
+clean:
+	rm out/*.conf
```

Comment:
this seems like a pretty simple makefile--have you considered just making them npm `script` entries?  The main downside of `make` (IMO) is that it actually requires either the XCode command line tools or brew to use--it doesn't come with OS X by default like, say, python does.  So you could get rid of the extra dependency by just integrating these into `package.json` and using npm as your task runner.

It'd also make it easier to run this on Windows but gosh I dunno why anyone would want to do that... 😁